DOKS-automatic-node-repair/README.md

```diff
@@ -1 +1,374 @@
-TBD
+# DigitalOcean Kubernetes Automatic Node Repair
+
+## Problem Description
+
+`DigitalOcean Kubernetes` (DOKS) cluster pool nodes can transition to an unhealthy state (`NotReady` or `Unknown`) by user will or by accident (overloading, deleting `kube-system` stuff, `kubelet` issues).
+
+## Solution:
+
+An auto-repair mechanism is needed in place so that the affected nodes from the cluster pool will be replaced automatically without user intervention.
+
+Possible ways of achieving this:
+ - [Digital Mobius](https://github.com/Qovery/digital-mobius)
+ - [Draino](https://github.com/planetlabs/draino) and [Kubernetes Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler)
+
+In this tutorial the main focus will be the [Digital Mobius](https://github.com/Qovery/digital-mobius) project. A simplified diagram of the internal flow (logic) is provided as well. Finally, we will have it deployed to a running `DOKS` cluster and observe how well it handles cluster nodes recycling.
```

Comment:
It'd be nice if we described why we prefer the Digital Mobius approach (e.g., simper to implement, better capabilities, etc.)

Reply:
Thanks for the suggestion. Fixed in the latest commit.